docs/guides/sap/odata-connector.md

```diff
@@ -0,0 +1,184 @@
+---
+
+The SAP OData Connector is a protocol and outbound [Connector](/components/connectors/introduction.md) that runs as a Docker image on the SAP Business Technology Platform (BTP).
+
+This Connector is designed to run in [hybrid mode](/guides/use-connectors-in-hybrid-mode.md), living in the customer's BTP sub-account in the Cloud Foundry environment.
+
+This Connector works with Camunda 8 SaaS, and utilizes BTP's [Destination](https://learning.sap.com/learning-journeys/administrating-sap-business-technology-platform/using-destinations) and [Connectivity](https://help.sap.com/docs/connectivity/sap-btp-connectivity-cf/what-is-sap-btp-connectivity) concepts to query a SAP system via both OData v2 and v4.
+
+:::note Important!
+This Connector is an alpha feature and available upon request. Visit [https://camunda.com/contact](https://camunda.com/contact) to contact us.
+:::
+
+## Quickstart
+
+For a standard overview of the steps involved in the SAP OData Connector, examine the diagram below:
+
+![OData steps](./img/odata-steps.png)
+
+## Prerequisites
+
+This Connector integrates into a standard BTP landscape and doesn't require any proprietary setup.
+
+However, a minimum set of SAP infrastructure requirements must be in place for the SAP OData Connector Docker image to run:
+
+- [Cloud Foundry CLI](https://github.com/cloudfoundry/cli) with [multiapps plugin](https://github.com/cloudfoundry/multiapps-cli-plugin) installed on the machine doing the deployment.
+- BTP subaccount with a [Cloud Foundry environment](https://discovery-center.cloud.sap/serviceCatalog/cloud-foundry-runtime?region=all) enabled and a [space created](https://help.sap.com/docs/btp/sap-business-technology-platform/create-spaces).
+- Minimum of [1GB storage quota, 2 GB runtime memory](https://help.sap.com/docs/btp/sap-business-technology-platform/managing-space-quota-plans).
+- [Entitlements](https://help.sap.com/docs/btp/sap-business-technology-platform/managing-entitlements-and-quotas-using-cockpit) for:
+  - [Connectivity Service](https://discovery-center.cloud.sap/serviceCatalog/connectivity-service?region=all), `lite` plan (to connect to the SAP is on-premises)
+  - [Destination Service](https://discovery-center.cloud.sap/serviceCatalog/destination?service_plan=lite&region=all&commercialModel=btpea), `lite` plan
+- One or more instance- or subaccount-level destinations, pointing to the SAP systems to communicate with.
+  ![sample BTP destination configuration](/Users/volker.buzek/git/camunda/camunda-docs/docs/guides/sap/images/btp-destination.png)
+
+:::warning
+Ensure `Additional Properties` are correctly set on the Destination.
+
+Here's a recommended set of properties:
+
+```json
+HTML5.DynamicDestination: true
+sap-client: <client/"Mandant" to work with on the SAP system>
+WebIDEEnabled: true
+WebIDESystem: <SAP system ID>
+WebIDEUsage: odata_gen
+```
+
+:::
```

Comment:
Can we move this out of a warning and into a standard part of the prerequisites?  It might give this section some room to breathe/add a little more explanation. What is "correctly"? Is this set only recommended, or is a version required to work?

Reply:
this is "recommended", the customers mileage may vary though